runtime/browser/cdn/browser-cdn-loader.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+const Loader = require('../../loader.js');
+const particle = require("../../particle.js");
+const DomParticle = require("../../dom-particle.js");
+
+module.exports = class BrowserLoader extends Loader {
+  constructor(urlMap) {
```

Comment:
> Re (1), the location of worker-entry is not discoverable and the urlMap is a convenient spot to configure that path.

I thought it would be a known relative path from ArcsLib.

> (3) is just convenient, especially today when these resource locations are fluid.
> 
> Then we shouldn't need another worker entry point.
> Hrm, not following you there, can you explain?

I meant that the only difference between worker-entry-cdn and worker-entry seems to be the loader.